schemas/ai34_det_counts.fbs

```diff
@@ -0,0 +1,11 @@
+// Schema for PulseImage, storing detector counts for eventmessage visualisation
+
+file_identifier "ai34";
+
+// NB, this is a sparse representation, 0 counts and their corresponding detector id are not included
+table PulseImage {
+    pulse_time : ulong;       // Time of source pulse, nanoseconds since Unix epoch (1 Jan 1970)
+    detector_id : [uint];     // Recorded detector ids. Corresponds directly by index with detection_count
+    detection_count : [uint]; // Total detection counts on each pixel from this pulse. Corresponds directly by index with detector_id
+}
```

Comment:
Could it make sense to also include a `source_name`? It would be optional anyway by flatbuffer's inherent design.

Reply:
Thanks for the suggestion. I can imagine that it could be useful in a debugging situation. I'm going to be deploying the producer for these in our production setup so I'll find out whether that is the case. I'd rather wait and gain experience of whether it is useful than add things on a "just in case" basis. It would be a non-breaking change to add it later anyway.